test/utility/Time.ts

```diff
@@ -0,0 +1,15 @@
+import Contracts, { TestTime } from '../../components/Contracts';
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+
+describe('Time', () => {
+    let time: TestTime;
+
+    beforeEach(async () => {
+        time = await Contracts.TestTime.deploy();
+    });
+
+    it('should return the time of the current block', async () => {
+        expect(await time.callStatic.realTime()).to.equal((await ethers.provider.getBlock('latest')).timestamp);
```

Comment:
Why is `callStatic` needed here?

Reply:
ecause each non-call transaction increments the current block number/time